test/EFCore.Relational.Tests/Metadata/RelationalEntityTypeAttributeConventionTest.cs

```diff
@@ -84,6 +93,7 @@ protected virtual ModelBuilder CreateConventionalModelBuilder()
             => RelationalTestHelpers.Instance.CreateConventionBuilder();
 
         [Table("MyTable", Schema = "MySchema")]
+        [Description("Test comment table")]
```

Comment:
It looks like you're referring to a table of comments.
```suggestion
        [Description("Test table comment")]
```